central/imagev2/datastore/store/postgres/store.go

```diff
@@ -69,6 +76,8 @@ type storeImpl struct {
 	keyFence           concurrency.KeyFence
 }
 
+// TODO(ROX-29941): Add scoping to all queries
```

Comment:
Should be there.  https://github.com/stackrox/stackrox/pull/16115

Reply:
Had to update `Get()` and `GetByIDs()` in the image store to use the framework instead of raw SQL for getting images. That should take care of search scoping and SAC. `Get()` and `GetByIDs()` also want to run multiple queries in a single transaction to get image, its components and CVEs. So I added the transaction into a context before passing it to framework. Take a look at those if that looks correct

https://github.com/stackrox/stackrox/pull/16137/files#diff-3159972457823a62ffb03c3045301c918e1713a503462249eca085938178cea8R595-R599

https://github.com/stackrox/stackrox/pull/16137/files#diff-3159972457823a62ffb03c3045301c918e1713a503462249eca085938178cea8R841-R844